modules/apps/client-extension/client-extension-web/src/main/java/com/liferay/client/extension/web/internal/portlet/CustomElementCETPortlet.java

```diff
@@ -162,6 +166,10 @@ private String[] _prepareURLs(long lastModified, String[] urls) {
 			if (!urls[i].startsWith("module:")) {
 				urls[i] = "nocombo:" + urls[i];
 			}
+
```

Comment:
The `if` above is looking for `module:` and/or adding `nocombo:` depending on what comes in `urls[i]`. That makes me think that your new `if` will fail under some conditions maybe.

Also: why "not startsWith"? Shouldn't it be "startsWith"? :thinking: